x-pack/plugins/infra/public/components/logging/log_entry_flyout/log_entry_actions_menu.tsx

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { EuiButtonEmpty, EuiContextMenuItem, EuiContextMenuPanel, EuiPopover } from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n/react';
+import React, { useCallback, useMemo, useState } from 'react';
+import url from 'url';
+
+import chrome from 'ui/chrome';
+import { InfraLogItem } from '../../../graphql/types';
+
+const UPTIME_FIELDS = ['container.id', 'host.ip', 'kubernetes.pod.uid'];
+
+export const LogEntryActionsMenu: React.FunctionComponent<{
+  logItem: InfraLogItem;
+}> = ({ logItem }) => {
+  const [hide, isVisible, show] = useVisibility();
+
+  const uptimeLink = useMemo(() => getUptimeLink(logItem), [getUptimeLink, logItem]);
```

Comment:
Would the reference of `getUptimeLink` ever change since it's defined in this file?

Reply:
No, it's just a defensive style in case a refactoring turns it into a prop. I could remove it.

Reply:
removed